src/AerialShip/SamlSPBundle/State/SSO/SSOStateStoreManager.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace AerialShip\SamlSPBundle\State\SSO;
+
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * This will manage the SSOState implementations.
+ *
+ * Class SSOStateStoreManager
+ * @package AerialShip\SamlSPBundle\State\SSO
+ */
+class SSOStateStoreManager implements SSOStateStoreInterface
```

Comment:
this should probably be named ``DoctrineStateStore`` as it is the doctrine-based implementation